### ISSUE TITLE
MoBi Toolbox for R - issues with 64 bit R

### ISSUE THREAD

**Issue by DanielMoj** [Open-Systems-Pharmacology/Forum#233]
Dear all,

I have some issue with installing and using the MoBi toolbox for R. My be you have some ideas on how to move on ...
I usually use R 64 bit... so I installed the MoBi toolbox on this 64 bit version. This does not seem to work (R error message: "package is not installed for 'arch=x64'")
Then, I tried to remove the toolbox package, and re-installed the toolbox using the 32 bit R version... However, now R is telling me that
_"package ‘MoBiToolboxForR’ was installed by an R version with different internals; it needs to be reinstalled for use with this R version"_

How to I correctly de-install the package installation for R 64 bit?? Someone got the same issue??

Thanks in advance
Daniel

**Comment by StephanSchaller** [comment]
Yes, you need to download the R-toolbox from the [esqLABS R-Toolbox repository](https://github.com/esqLABS/R-Toolbox/releases) to have it working with 3.5.
The one coming with the OSPSuite is only compatible with R < 3.5 (@msevestre or has this been updated for the OSPS R-Toolbox repository?).


Next release (OSPS 7.4) will have 64Bit support for R (right @msevestre ?).

Best,
Stephan

**Comment by msevestre** [comment]
Hi @DanielMoj 

I cannot comment on this parallel fork developed by @StephanSchaller as I was not aware of its existence until now.

In the meantime, we have indeed move to x64 bit. The version is not released yet but has been used successfully as far as I know. You can download the latest R version, **again not officially released**, supporting x64 bit [here](https://ci.appveyor.com/project/open-systems-pharmacology-ci/r-toolbox/branch/develop/artifacts).

Hope that helps,

**Comment by DanielMoj** [comment]
Thank you @StephanSchaller and @msevestre !!
I will give esqLABS toolbox version a first try! 

@msevestre Do you have an estimate on when the x64 bit toolbox is going to be released?

Best
Daniel

**Comment by DanielMoj** [comment]
@StephanSchaller may be I misunderstood... is the R-toolbox from the esqLABS R-Toolbox repository working with the 64 bit version of R? 

In the esqLABS repository description it is saying: _"As for now, only the 32-bit version of the OSPS-Toolbox is available. Make sure the 32-bit (also called x86) version of R is installed and used when working the the OSPS-Toolbox"_

**Comment by StephanSchaller** [comment]
No it is 32 Bit. Just like the released OSPS. But it works with R3.5

**Comment by msevestre** [comment]
@DanielMoj Estimate would be November 2018
